docker/docker-compose-core.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.3'
+
+services:
+  core:
+    restart: on-failure
+    container_name: codesearch_core
+    image: quay.io/aelve/codesearch-core:latest
+    external_links:
+    - postgres_1
+    volumes:
+    - /mnt/vol/data:/data
+    - /mnt/vol/index:/index
+    - /mnt/vol/logs:/logs
+    command: >
+      sh -c "make tables && \
+            make download-haskell && \
```

Comment:
You can group them together to improve readability:

```
make download-haskell update-haskell index-haskell && \
```

Reply:
Ok.